<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputTab</class>
 <widget class="QWidget" name="OutputTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>316</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Outputs</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="generate">
        <property name="text">
         <string>Write Tiles</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
        <widget class="QPushButton" name="outputNew">
         <property name="text">
          <string>New</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/resources/plus.svg</normaloff>:/resources/plus.svg</iconset>
         </property>
        </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="outputCopy">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Copy</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="outputDelete">
          <property name="text">
           <string>Delete</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/resources/trashicon.svg</normaloff>:/resources/trashicon.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="outputList"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tilePage">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Output Directory</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="tileOutputPathUseProjectDir">
            <property name="text">
             <string>Output In Project Directory</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="tilePath">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="maximumSize">
               <size>
                <width>23</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Min Zoom</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QSpinBox" name="minZoom">
         <property name="maximum">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Max Zoom</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QSpinBox" name="maxZoom">
         <property name="maximum">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QCheckBox" name="tileCreate2xRes">
         <property name="text">
          <string>Create 2x Resolution </string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Tile Size</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QComboBox" name="tileSize"/>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="group">
         <property name="title">
          <string>Bounding Box</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="2">
           <widget class="QDoubleSpinBox" name="tileLatBottom">
            <property name="decimals">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QDoubleSpinBox" name="tileLongRight">
            <property name="decimals">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="tileLongLeft">
            <property name="decimals">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Longitude</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QDoubleSpinBox" name="ttileLatTop">
            <property name="decimals">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Latitude</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="tileCreate1xRes">
         <property name="text">
          <string>Create Standard Resolution</string>
         </property>
        </widget>
       </item>
      </layout>
      </widget>
      <widget class="QWidget" name="imagePage">
       <layout class="QFormLayout" name="formLayout_image">
        <item row="0" column="0" colspan="2">
         <widget class="QGroupBox" name="imageOutputDir">
          <property name="title">
           <string>Output Directory</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_image">
          <item>
           <widget class="QCheckBox" name="imageOutputPathUseProjectDir">
            <property name="text">
             <string>Output In Project Directory</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_image">
            <item>
             <widget class="QLineEdit" name="imagePath">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="imagePathBrowse">
              <property name="maximumSize">
               <size>
                <width>23</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="imageWidthLabel">
          <property name="text">
           <string>Width</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QSpinBox" name="imageWidth">
          <property name="maximum">
           <number>4096</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="imageHeightLabel">
          <property name="text">
           <string>Height</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QSpinBox" name="imageHeight">
          <property name="maximum">
           <number>4096</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="imageBoundingBox">
          <property name="title">
           <string>Bounding Box</string>
          </property>
           <layout class="QGridLayout" name="gridLayout_image">
           <item row="3" column="2">
            <widget class="QDoubleSpinBox" name="imageLatBottom">
             <property name="decimals">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QDoubleSpinBox" name="imageLongRight">
             <property name="decimals">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QDoubleSpinBox" name="imageLongLeft">
             <property name="decimals">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_image_long">
             <property name="text">
              <string>Longitude</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QDoubleSpinBox" name="imageLatTop">
             <property name="decimals">
              <number>5</number>
             </property>
            </widget>
           </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_image_lat">
              <property name="text">
               <string>Latitude</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0" colspan="4">
             <widget class="QPushButton" name="imageBboxSelect">
              <property name="text">
               <string>Choose on Map...</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
       </layout>
      </widget>
      <widget class="QWidget" name="blankPage"/>
      </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
